core/src/commonMain/kotlin/com/ustadmobile/core/controller/PersonAccountEditPresenter.kt

```diff
@@ -0,0 +1,110 @@
+package com.ustadmobile.core.controller
+
+import com.ustadmobile.core.account.UnauthorizedException
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.impl.UstadMobileSystemImpl
+import com.ustadmobile.core.util.ext.putEntityAsJson
+import com.ustadmobile.core.view.PersonAccountEditView
+import com.ustadmobile.core.view.UstadEditView.Companion.ARG_ENTITY_JSON
+import com.ustadmobile.core.view.UstadView.Companion.ARG_ENTITY_UID
+import com.ustadmobile.core.view.UstadView.Companion.ARG_SERVER_URL
+import com.ustadmobile.door.DoorLifecycleOwner
+import com.ustadmobile.door.doorMainDispatcher
+import com.ustadmobile.lib.db.entities.Person
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withTimeoutOrNull
+import kotlinx.serialization.json.Json
+import org.kodein.di.DI
+import org.kodein.di.instance
+
+
+class PersonAccountEditPresenter(context: Any,
```

Comment:
I think for this to stick to the crud pattern we should add another object which holds the data that is going to be filled in. E.g. you could add:
```
class PersonWithAccount(val currentPassword: String, val newPassword: String, val passwordConfirmed: String)
```

The pattern we are following is that the presenter gives the object to the view. The object should generally have the properties that are going to be manipulated in the view.